### PR TITLE
Initial implementation of Gondor claiming Harnendor

### PR DESCRIPTION
**Pull request by kvopper**
Sorry for mistakes in commit names

### CODE REVIEW

File: decisions/Gondor.txt [hunk 1]
```diff
@@ -1789,6 +1789,33 @@ political_decisions = {
...
 	}
 	
+	claim_harnendor = {
+		picture = "harnendor_map"
+		potential = {
+			tag = GON
+			HRD = { all_core = { owned_by = GON } }
+			NOT = {
+				has_global_flag = harnendor_claimed_GON
+			}
```
Comment (SirRunner) [inline decisions/Gondor.txt:1799]: We're going to want to tie this into their military recruitment flavor. Gondor could get all of Harondor's cores using normal mechanics, but it wouldn't really make sense if they still had low_military_recruitment -- their expansion flavor is locked behind NOT having that modifier...

`NOT = { has_country_modifier = low_military_recruitment }`

File: decisions/Gondor.txt [hunk 2]
```diff
@@ -1789,6 +1789,33 @@ political_decisions = {
...
+		
+		effect = {
+			prestige = 10
+			badboy = 3
+			HRN = {
+				all_core = {
+					add_core = GON
+					add_core = HRD
+				}
+			}
```
Comment (SirRunner) [inline decisions/Gondor.txt:1814]: In reference to our Discord conversation earlier, HRN cores != the Harnendor that Gondor once owned. 
 - HRN = the Haradrim tribe (that I happened to give a matching name to) 
 - HAR_286 = the region that Gondor calls Harnendor

HAR_286 = { all_core = { add_core = GON } }

We should also probably accommodate for the possibility of Harondor cores being removed (which I think does happen if the player follows flavor)
```
# Adds Harondor cores to Harnendor (HAR_286) if Harondor cores haven't been removed
random_owned = {
    limit = { province_id = 249 }
    owner = { HAR_286 = { all_core = { add_core = HRD} } }
}
```

I believe that should do the trick

File: localisation/decisions.csv [hunk 3]
```diff
@@ -327,6 +327,8 @@ annex_south_gondor_title;Annex South Gondor;x
 annex_south_gondor_desc;Finally, we control all of South Gondor. It is time we abolish the fiefdom government, and merge it with the realm's government.;x
 accept_harnedan_title;The Peoples of Harondor;x
 accept_harnedan_desc;Many inhabitants of South Gondor are of both Gondorian and Haradrim descent - they are the sons of the Gondorian settlers who came to South Gondor in the past. When Gondor left the region, the settlers stayed, and were integrated into the local Haradrim society. They are rightfully descendants of Gondor, and should be recognized as such.;x
+claim_harnendor_title;Claim Harnendor;x
+claim_harnendor_desc;In the past, the territory south of the Harnen river known as N�nzayan was Gondorian, and it's culture quite similar to that of Harondor. With our renewed presence in the area, we can reassert our claim to this land.;x
```
Comment (SirRunner) [inline localisation/decisions.csv:331]: Nînzayan is the Haradrim name for Harnendor (the region)

Comment (kvopper) [reply localisation/decisions.csv:331]: Should've clarified that it's the Haradrim who call it Nînzayan

**Pull request closed without merging**